utils/misc/misc.go

```diff
@@ -47,10 +47,13 @@ import (
 	"github.com/thoas/go-funk"
 )
 
-var AppStartTime int64
-var errorStorePath string
-var reservedFolderPaths []*RFP
-var jsonfast = jsoniter.ConfigCompatibleWithStandardLibrary
+var (
+	AppStartTime        int64
+	errorStorePath      string
+	reservedFolderPaths []*RFP
+	jsonfast            = jsoniter.ConfigCompatibleWithStandardLibrary
+	notifyOnce          sync.Once
```

Comment:
Under which scenario do we get more than 1 bugsnag notifications per crash?

Do we need a different `sync.Once` per team?

Reply:
not sure, but I don't think we need more than 1 bugsnag notification per crash. My line of thought is, currently we can have two values for `Team` `Warehouse` & `Core`. Both, seems completely independent.